kolibri/plugins/user_profile/assets/src/views/ChangeFacility/ConfirmAccount/index.vue

```diff
@@ -46,23 +50,26 @@
     inject: ['changeFacilityService', 'state'],
     computed: {
       targetFacility() {
-        return this.state.value.targetFacility;
+        return get(this.state, 'value.targetFacility');
```

Comment:
The smoke[ test I added here](https://github.com/learningequality/kolibri/pull/9562/files#diff-fd0b011ddc9a568a5c730be0271e1f026140c6a528245f4e65ee5de91bb7ca4eR28-R31) would fail if I didn't do this. The purpose of that test is to ensure that the component can run without any runtime errors even before it receives data from other application layers. Making assumptions that some data is available immediately can be tricky in async reactive environments and creates hidden dependencies.

Regarding using lodash `get`, I learned that from someone's PR recently for accessing nested properties so that I don't need to write `if (this.state && this.state.value`). No problem to remove it, it's a personal preference. What's the problem with `import`?

Reply:
ok, thanks for the explanation. There's no problem with the import, I just like to avoid adding external functions if it can be done in plain JS but after reading your comment it seems it's needed.